src/workflows/Case.workflow

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workflow xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
Why this workflow file is included in this PR? Is this related to any assignment? If not please remove it.